### DISCUSSION TITLE
FRAME_SIZE_ERROR-Open telemetry agent

### DISCUSSION THREAD

**Discussion by manvitha9347** [open-telemetry/opentelemetry-java#4422] [Q&A]
I am trying to send traces collected by open telemetry agent from spring boot code to tempo via grafana agent
my tempo is configured to listen at ports 4317,4318
grafana agent is configured for otlp as well
i am facing this issue when i try to hit my api end point

vm args:
-javaagent:C:\opentelemetry-javaagent.jar 
-Dotel.metrics.exporter=none 
-Dotel.traces.exporter=otlp
-Dotel.exporter.otlp.traces.endpoint=http://localhost:4318
-Dotel.resource.attributes="service.name=myotelapp"

ERROR io.opentelemetry.exporter.otlp.internal.grpc.OkHttpGrpcExporter - Failed to export spans. The request could not be executed. Full error message: FRAME_SIZE_ERROR: 4740180

**Comment by jkwatson** [comment]
This doesn't sound like an issue with the java exporter, but with however you've configured your tempo collector.

**Reply by trask** [reply]
@manvitha9347 maybe https://github.com/grpc/grpc-java/issues/8164#issuecomment-838877435 will help you?
